examples/bip32.rs

```diff
@@ -58,7 +58,7 @@ fn main() {
     let public_key = xpub.derive_pub(&secp, &vec![zero, zero])
                          .unwrap()
                          .public_key;
-    let address = Address::p2wpkh(&public_key, network).unwrap();
+    let address = Address::p2wpkh(&ecdsa::PublicKey { compressed: true, key: public_key }, network).unwrap();
```

Comment:
I'm a little tempted to just impl `From<secp256k1::PublicKey>` for `ecdsa::PublicKey` which does this.

Reply:
There is already a `new` function that defaults to compressed IIRC.